frontend/public/auth.html

```diff
@@ -3,9 +3,85 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Auth page</title>
+    <link rel="stylesheet" href="/styles.css">
+    <title>Login</title>
 </head>
-<body>
-    <h1>Login/Sign up</h1>
+<body class="login">
+    <table class="loginForm">
+        <tr>
+            <td><label for="usernamefield">Username:</label></td>
+            <td><input type="text" class="loginPageText" id="usernamefield" placeholder="Enter username"></td>
+        </tr>
+        <tr>
+            <td><label for="passwordfield">Password:</label></td>
+            <td><input type="password" class="loginPageText" id="passwordfield" placeholder="Enter password"></td>
+        </tr>
+    </table>
+    <div class="lsButtons">
+        <input type="button" class="lsButton" id="loginButton" name="login" value="Login" onclick="loginUser()">
+        &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
I'm unfamiliar with the usages of non-breaking spaces. Would padding or margin not have worked here?